migrateToAutomaticPackageRestore.ps1

```diff
@@ -7,6 +9,7 @@ $listOfBadStuff = @(
 	"\s*<Import Project=""\$\(SolutionDir\)\\\.nuget\\NuGet\.targets"".*?/>",
 	"\s*<Target Name=""EnsureNuGetPackageBuildImports"" BeforeTargets=""PrepareForBuild"">(.|\n)*?</Target>"
 	"\s*<RestorePackages>\w*</RestorePackages>"
+    "\s*<SolutionDir Condition=""\$\(SolutionDir\) == '' Or \$\(SolutionDir\) == '\*Undefined\*'"">\.\.\\</SolutionDir>"
```

Comment:
I've updated the code to optionally remove the SolutionDir property based on a command line parameter.